include/miopen/miopen.h

```diff
@@ -5862,6 +5862,101 @@ MIOPEN_EXPORT miopenStatus_t miopenGroupNormForward(miopenHandle_t handle,
 // CLOSEOUT groupnorm DOXYGEN GROUP
 #endif
 
+#ifdef MIOPEN_BETA_API
+// InstanceNorm APIs
+/** @addtogroup instancenorm
+ *
+ *  @{
+ */
+/*! @brief Execute a instancenorm forward layer
+ *
+ * @param handle         MIOpen handle (input)
+ * @param inputDesc      Tensor descriptor for data input tensor input (input)
+ * @param input          Data tensor input (input)
+ * @param outputDesc      Tensor descriptor for data input tensor output (output)
+ * @param oputput          Data tensor out (output)
+ * @param weightDesc      Tensor descriptor for data input tensor weight (input)
+ * @param weight          Data tensor weight (input)
+ * @param biasDesc      Tensor descriptor for data input tensor bias (input)
+ * @param bias          Data tensor bias (input)
+ * @param meanInDesc      Tensor descriptor for data input tensor running mean in (input)
+ * @param meanIn          Data tensor mean in (input)
+ * @param varInDesc      Tensor descriptor for data input tensor runnign var in (input)
```

Comment:
two typos (runnign) here.

Reply:
resolved in [13a3551](https://github.com/moreh-dev/MIOpen/pull/31/commits/13a35518664a45f8642f4cff1bfa7c71e4296a12)